workshop/session_3/11_biopython_proteins.ipynb

```diff
@@ -0,0 +1,957 @@
+      "text/plain": [
+       "<Figure size 480x320 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "plt.rcParams['figure.dpi'] = 80\n",
+    "\n",
+    "plt.plot(hydrophobicity)\n",
+    "plt.title(\"Hydrophobicity per sequence region using window size {}\".format(window_size))\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": false
+   },
+   "source": [
+    "## BLAST\n",
+    "Basic Local Alignment Search Tool allows finding similar regions across proteins and retrieving the most similar ones.\n",
+    "\n",
+    "**Biopython** provides tools for both local BLAST tools (e.g. those normally run on a command line) as well as remote computation services. In this section, we will use NCBI BLAST cloud services.\n",
+    "\n",
+    "*Note: running BLAST is computationally heavy, especially so with large databases of sequences, expect any calls to take at least several minutes.*\n",
+    "\n",
+    "**Biopython** Blast module contains two classes:\n",
+    " - `NCBIWWW` - to issue queries to the remote server\n",
+    " - `NCBIXML` - to convert results (in XML format) to an object that can be easily used in the code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 171,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from Bio.Blast import NCBIWWW, NCBIXML\n",
+    "# This code will take several minutes to run as it executes a BLAST search in NCBI cloud environment.\n",
+    "query_handle = NCBIWWW.qblast(\"blastp\", database=\"nr\", sequence=cftr_aa.seq)\n",
```

Comment:
Can you add a comment explaining the parameters briefly? i.e 'nr' indicates a non-redundant sequence database

Reply:
Good call, added in-line comments.